Trakt/Helpers/LibraryManagerEventsHelper.cs

```diff
@@ -81,17 +81,17 @@ private async void OnQueueTimerCallback(object state)
             }
             catch (Exception ex)
             {
-                _logger.ErrorException("Error in OnQueueTimerCallbackInternal", ex);
+                _logger.LogError("Error in OnQueueTimerCallbackInternal", ex);
```

Comment:
This would probably eat the exception, at least according to https://stackoverflow.com/a/47771450 you should do
```suggestion
                _logger.LogError(ex, "Error in OnQueueTimerCallbackInternal");
```